wxCompile.nim

```diff
@@ -0,0 +1,102 @@
+# additional compiler and linker instructions to include wxWidgets binaries in Nim
+# Fredrik Høisæther Rasch (@couven92), 2017
+
+import ospaths
+
+when defined(wxWidgetsPath):
+  const wxWidgetsPath {.strdefine.}: string = nil
```

Comment:
This shouldn't compile with devel.

Reply:
@Araq, why not?

Reply:
Because ``nil`` stopped being an allowed value for strings.